packages/devtools_app_shared/lib/src/ui/theme/theme.dart

```diff
@@ -300,6 +300,7 @@ double get inputDecorationElementHeight => scaleByFontFactor(20.0);
 // Padding / spacing constants:
 const extraLargeSpacing = 32.0;
 const largeSpacing = 16.0;
+const extraSpacing = 14.0;
```

Comment:
nit: instead of adding another spacing const to this public package, maybe just declare this value in a local variable where it is used

Reply:
Done